packages/fast-components-react-base/src/select/select.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "title": "Select",
+    "description": "A select component's schema definition.",
+    "type": "object",
+    "id": "select",
```

Comment:
This will need to be brought inline with our new ID policy, per @chrisdholt work in the v4 PR, this should be `@microsoft/fast-components-react-msft/select`

Reply:
done